tests/1_challangeExit.js

```diff
@@ -0,0 +1,98 @@
+const { Tx, Input, Output, Outpoint, Period, helpers } = require('leap-core');
+
+const challenge = require('../scripts/challenge');
+
+const mintAndDeposit = require('./actions/mintAndDeposit');
+const { sleep, unspentForAddress } = require('../src/helpers');
+
+module.exports = async function(contracts, nodes, accounts, web3) {
+
+  const minter = accounts[0].addr;
+  const alice = accounts[2].addr;
+  const alicePriv = accounts[2].privKey;
+  const challenger = accounts[3].addr;
+  const challengerPriv = accounts[3].privKey;
+  const zzz = accounts[9].addr;
+  const amount = 10000000;
+  const color = 0;
+  const node = nodes[0];
+
+  await mintAndDeposit(alice, amount, minter, contracts.token, contracts.exitHandler);
+  await sleep(8000);
+
+  let state = await node.getState();
+  console.log(state);
+  let aliceUnspent = unspentForAddress(state.unspent, alice, color).map(u => ({
+    output: u.output,
+    outpoint: Outpoint.fromRaw(u.outpoint),
+  }));
+  let inputs = [new Input(aliceUnspent[0].outpoint)];
+  let outputs = [new Output(amount, alice.toLowerCase(), color)];
+  let firstTransfer = Tx.transfer(inputs, outputs).signAll(alicePriv);
```

Comment:
can be replaced with `Tx.transferFromUtxos` ?

Reply:
Okay